pkg/arvo/lib/naive.hoon

```diff
@@ -402,10 +382,12 @@
 ::
 ++  ud-to-ascii
   |=  n=@ud
-  ^-  @t
-  ?~  n
-    *@t
-  (cat 3 $(n (div n 10)) (add '0' (mod n 10)))
+  ?~  n  '0'
```

Comment:
The previous implementation didn't handle `0`. If all argument are known to be non-zero, this line can be deleted.